psi4/src/read_options.cc

```diff
@@ -247,6 +247,18 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
     /*- How many NOONS to print -- used in libscf_solver/uhf.cc and libmints/oeprop.cc -*/
     options.add_str("PRINT_NOONS", "3");
 
+    ///MBIS Options (libmints/oeprop.cc)
+
+    /*- Maximum Number of MBIS Iterations -*/
+    options.add_int("MBIS_MAXITER", 500);
+    /*- MBIS Convergence Criteria -*/
+    options.add_double("MBIS_D_CONVERGENCE", 1.0e-8);
+    /*- MBIS Number of Radial Points -*/
+    /*- Additional Radial Points may be needed for Heavier Atoms (200-300) like Zinc -*/
+    options.add_int("MBIS_RADIAL_POINTS", 99);
+    /*- MBIS Number of Spherical Points -*/
+    options.add_int("MBIS_SPHERICAL_POINTS", 350);
```

Comment:
wasn't the plan to use normal dft grid by default (75, 302), then warn for nonphysical integration so user could retry with tighter grid?

Reply:
Based on tests I ran on the Benzene-Naphthalene Series, I discovered that 99/350 is indeed a better default than 75/302 or even 99/302, based on looking at the error growth

Electron Errors:

75/302:
Benzene: 2E-5
Naphthalene: 7E-5
Anthracene: 9E-5
Tetracene: 12E-5
Pentacene: 14E-5

99/302:
Benzene: 2E-5
Naphthalene: 7E-5
Anthracene: 10E-5
Tetracene: 12E-5
Pentacene: 14E-5

99/350:
Benzene: 1E-5
Naphthalene: 3E-5
Anthracene: 3E-5
Tetracene: 2E-5
Pentacene: 2E-5

Reply:
smaller grid errors are larger but not troublesomely so -- less than an order of magnitude. do the charges change appreciably (>0.02) over the different grid sizes? (75, 302) looking good to me.

Reply:
All atomic charges are within 0.001 electrons of each other on tetracene when compared on a 75/302 grid and a 99/350 grid